utils/forest_shim/src/randomness.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+pub use fvm_shared::randomness::Randomness as Randomness_v2;
+pub use fvm_shared3::randomness::Randomness as Randomness_v3;
```

Comment:
Those should not be re-exported unless necessary. Please look at @creativcoder PR https://github.com/ChainSafe/forest/pull/2471/files#diff-50b0624229c10e65381e9ecd7fa359b4b07dc469b5e4ad43f637a6e27abd80f1. Also, some small set of tests like there would help.

Reply:
added